test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -323,6 +324,21 @@ class CBasicTestLoad
             == TEST_SKIPPED_ITSELF)
             return 0; // skip test - not applicable
 
+        if (CBasicTestMemOrderScope<HostAtomicType, HostDataType>::DataType()
+                ._type
+            != TYPE_ATOMIC_HALF)
```

Comment:
Please check this especially.  Should this be testing for `!= TYPE_ATOMIC_HALF`, or `== TYPE_ATOMIC_HALF`?

Reply:
Yes, copy/paste error, went through because my device has `...FP_ATOMIC_LOAD_STORE_EXT` capabilities available. Thanks.